app/javascript/mastodon/actions/compose.js

```diff
@@ -1,4 +1,5 @@
 import api from '../api';
+import axios from 'axios';
```

Comment:
Oh I get it now

`import axios, { CancelToken } from 'axios';`

should work instead of `let CancelToken`